iceoryx_posh/test/integrationtests/test_popo_chunk_building_blocks.cpp

```diff
@@ -54,7 +54,7 @@ struct ChunkDistributorConfig
 
 struct ChunkQueueConfig
 {
-    static constexpr uint64_t MAX_QUEUE_CAPACITY = iox::MAX_SUBSCRIBER_QUEUE_CAPACITY;
+    static constexpr uint64_t MAX_QUEUE_CAPACITY = NUM_CHUNKS_IN_POOL / 3;
```

Comment:
@budrus this time I haven't had luck. Hope this make it more robust. The `MAX_QEUEU_CAPACITY` is now the same like before, just the amount of chunks in the mempools are now three times as high

Reply:
IIRC @dkroenke has pinged @jpsamper2009 for dedicated hardware already

Reply:
The test runs out of chunks again. This should not be possible. I don't understand why it fails this way

Reply:
Hmm. Maybe we do not release the chunks we cannot push in a full FiFo. Normally we only use overflowing queues for chunk communication